frontends/web/src/api/bitbox02.ts

```diff
@@ -126,6 +126,13 @@ export const createBackup = (deviceID: string): Promise<void> => {
     });
 };
 
+export const restoreBackup = (
+  deviceID: string,
+  selectedBackup: string,
+): Promise<FailResponse | SuccessResponse> => {
```

Comment:
There seem to be another error that doesn't return success: false, but I am not sure

https://github.com/digitalbitbox/bitbox-wallet-app/blob/82b3291732d0e5030de43f1217caeadb2d84f91b/backend/devices/bitbox02/handlers/handlers.go#L206-L215

Reply:
Yes, you're right. If e.g. the backup ID is missing it returns `null`.

Reply:
thanks I'll update the PR and rebase the others.

Reply:
Hey, I tested it because I wanted to be sure. Apparently, the error is translated, so that if the backup id is missing the frontend result is actually not `null`, but a struct with a error field: like `{"error":"EOF"}`.
Sorry for the mistake :pray:

Reply:
I noticed that there is a 2nd translated error https://github.com/digitalbitbox/bitbox-wallet-app/blob/bd06af77e63dd84293dc9a3693164551cf0db660/frontends/web/src/locales/en/app.json#L122-L125

but e200 is not used for BB02 it only uses generic backup restore error

https://github.com/digitalbitbox/bitbox-wallet-app/blob/bd06af77e63dd84293dc9a3693164551cf0db660/frontends/web/src/routes/device/bitbox02/backups.tsx#L80  

in BB01 component translates it correctly:
https://github.com/digitalbitbox/bitbox-wallet-app/blob/bd06af77e63dd84293dc9a3693164551cf0db660/frontends/web/src/routes/device/bitbox01/restore.tsx#L101

I think it makes sense to clean this up so that the api error alway return FailResponse with "success", "code" and "message" fields, (what maybeError does).

Reply:
Opened a separated PR to fix the endpoint: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1933